api/pb/api.proto

```diff
@@ -6,14 +6,30 @@ option java_package = "io.textile.threads_grpc";
 option java_outer_classname = "Threads";
 option objc_class_prefix = "THRDS";
 
-message NewDBRequest {}
+message NewDBRequest {
+    string dbID = 1;
+}
 
-message NewDBReply {
-    string ID = 1;
+message NewDBFromAddrRequest {
+    string dbAddr = 2;
+    bytes followKey = 3;
+    bytes readKey = 4;
+}
+
+message NewDBReply {}
+
+message GetDBInfoRequest {
+    string dbID = 1;
+}
+
+message GetDBInfoReply {
+    repeated string addresses = 1;
+    bytes followKey = 2;
+    bytes readKey = 3;
 }
 
 message NewCollectionRequest {
-    string DBID = 1;
+    string dbID = 1;
```

Comment:
cool thanks for changing all these.

cc @andrewxhill this should make the generated js/ts code better